integration/kubernetes/singleMaster-localJournal/alluxio-configmap.yaml.template

```diff
@@ -36,7 +36,7 @@ data:
   ALLUXIO_WORKER_JAVA_OPTS: |-
     -Dalluxio.worker.bind.host=0.0.0.0  -Dalluxio.worker.data.server.domain.socket.address=/opt/domain -Dalluxio.worker.data.server.domain.socket.as.uuid=true -Dalluxio.worker.memory.size=2G -Dalluxio.worker.rpc.port=29999 -Dalluxio.worker.web.port=30000 -Dalluxio.worker.hostname=${ALLUXIO_WORKER_HOSTNAME} -Dalluxio.worker.tieredstore.levels=1 -Dalluxio.worker.tieredstore.level0.dirs.mediumtype=MEM -Dalluxio.worker.tieredstore.level0.dirs.path=/dev/shm -Dalluxio.worker.tieredstore.level0.dirs.quota=1G -Dalluxio.worker.tieredstore.level0.watermark.high.ratio=0.95 -Dalluxio.worker.tieredstore.level0.watermark.low.ratio=0.7 
   ALLUXIO_JOB_WORKER_JAVA_OPTS: |-
-    -Dalluxio.job.worker.rpc.port=30001  -Dalluxio.job.worker.data.port=30002 -Dalluxio.job.worker.web.port=30003 -Dalluxio.worker.hostname=${ALLUXIO_WORKER_HOSTNAME}
+    -Dalluxio.job.worker.rpc.port=30001  -Dalluxio.job.worker.data.port=30002 -Dalluxio.job.worker.web.port=30003 -Dalluxio.worker.hostname=${ALLUXIO_WORKER_HOSTNAME} 
```

Comment:
since these are simply a space-separated list, is it possible for us to simply split this into a key-value format in yaml and run some post-processing to generate the string needed when we pass all of these values to the containers? I think it is harder to maintain and more prone to errors like this

Reply:
Logged https://github.com/Alluxio/alluxio/issues/11081
This should be a low-hanging fruit.